static/js/apps/eval_retrieval_generation/app.tsx

```diff
@@ -143,6 +148,34 @@ export function App(props: AppPropType): JSX.Element {
     });
   };
 
+  const loadMetadata = (doc: GoogleSpreadsheet, allHeader: HeaderInfo) => {
+    const sheet = doc.sheetsByTitle[DC_METADATA_SHEET];
+    const header = allHeader[DC_METADATA_SHEET];
+    const loadPromises = [];
+    for (const col of [METADATA_KEY_COL, METADATA_VAL_COL]) {
+      loadPromises.push(
+        sheet.loadCells({
+          startColumnIndex: header[col],
+          endColumnIndex: header[col] + 1,
+        })
+      );
+    }
+    const numRows = sheet.rowCount;
+    Promise.all(loadPromises).then(() => {
+      for (let i = 1; i < numRows; i++) {
+        const metadataKey = sheet.getCell(i, header[METADATA_KEY_COL]).value;
+        if (metadataKey === METADATA_KEY_TYPE) {
+          setEvalType(
+            sheet.getCell(i, header[METADATA_VAL_COL]).value as EvalType
+          );
+          return;
+        }
+      }
+      // by default, set eval type as RIG
+      setEvalType(EvalType.RIG);
```

Comment:
maybe raise exception just in case this field is not populated? (old doc, or new type..)

Reply:
added as an alert